Doc/library/random.rst

```diff
@@ -326,7 +326,7 @@ be found in any statistics text.
 
 .. function:: gauss(mu, sigma)
 
-   Gaussian distribution.  *mu* is the mean, and *sigma* is the standard
+   Normal distribution.  *mu* is the mean, and *sigma* is the standard
```

Comment:
Perhaps add something like, "Normal distribution, also called the Gaussian distribution."

Since the function is called "gauss", the docs should at least reference the name.

Reply:
Would you suggest having that also in random.normalvariate or only in random.gauss?